osf/models/registrations.py

```diff
@@ -263,6 +275,11 @@ def date_withdrawn(self):
     def withdrawal_justification(self):
         return getattr(self.root.retraction, 'justification', None)
 
+    def require_approval(self, *args, **kwargs):
+        super().require_approval(*args, **kwargs)
```

Comment:
I don't think this is actually being used on the super class, better move the whole thing down here.